### PR TITLE
[Xcode13.2] Bump to Xcode 13.2 RC

### PR DESCRIPTION
**Pull request by dalexsoto**
Breaking changes addressed for legacy (Legacy API diff seems happy)
* HomeKit
* CallKit
* CoreLocation

Include all of [Xcode13.2 changes](https://github.com/xamarin/xamarin-macios/wiki/xcode13.2-Bindings-Status) but CHIP and MetalPerformanceShadersGraph.

### CODE REVIEW

**Comment by chamons** [comment]
Looks reasonable, but I've been out of binding practice for awhile so only tiny 👍.

My biggest concern here is API compat, which we'll have to lean on API diff + folks here.

**Review by mandel-macaque — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
There are also a few breaking changes:
Screen Shot 2021-12-07 at 10 58 02
Screen Shot 2021-12-07 at 10 54 07
Screen Shot 2021-12-07 at 10 51 22

File: src/corelocation.cs [hunk 1]
```diff
@@ -48,11 +48,12 @@ public enum CLProximity : long {
 		Far
 	}
 
+#if !MONOMAC
```
[the remark this answers is not in the document]
Comment (dalexsoto) [reply src/corelocation.cs:51]: I think the `ErrorDomain` was still generated I can double check this in the morning

**Comment by dalexsoto** [comment]
@rolfbjarne I think I have addressed all the feedback and also updated to Xcode 13 RC which has only one change on intents

**Comment by dalexsoto** [comment]
- known issue - [Bug17793](https://github.com/xamarin/maccore/issues/2443)
- xtro is fixed
- known issue - [*/Mac Catalyst/*: TimedOut](https://github.com/xamarin/maccore/issues/2414)

due to time constrains ✅

**Pull request merged**